tests/bsd/color.sh

```diff
@@ -0,0 +1,9 @@
+ORI_LSCOLORS="$LSCOLORS"
```

Comment:
You don't need to save these.  Each test case runs in a subshell, so environment changes won't affect other tests.

Also, `tests/bsd` is for features that are part of BSD system `find` implementations.  Since `-color` is a `bfs` extension, this test needs to go under `tests/bfs`, possibly something like `tests/bfs/color_bsd.sh`.